database/index.js

```diff
@@ -45,11 +44,20 @@ const createExpReviews = (expId, reviewsData, callback) => {
   });
 };
 
-const updateExpReviews = (reviewId, updateData, callback) => {
-  const reviews = { ...updateData };
+const getExpReview = (reviewId, callback) => {
+  const query = 'SELECT * FROM reviews WHERE id = ?';
+  dbConnection.query(query, reviewId, (err, response) => {
+    if (err) {
+      callback(err);
+    } else {
+      callback(null, response);
+    }
+  });
+};
+const updateExpReview = (reviewId, updateData, callback) => {
   const updateQuery = `UPDATE reviews SET ? WHERE id=${reviewId}`;
   const selectReviewQuery = `SELECT * FROM reviews WHERE id=${reviewId}`;
```

Comment:
Sorry If I missed something, but where are you using `selectReviewQuery`?

Reply:
Its the second query after the update query. Theres a way to expand the codes that were not changed (somewhere on the left). I think these PR only shows changes made so expanding will help you follow the flow of the code.